src/modules/economy/components/AccountCard.tsx

```diff
@@ -78,23 +63,70 @@ export const AccountCard: React.VFC<AccountCardProps> = ({ account }) => {
   }
 
   return (
-    <Card>
-      <Card.Section m="xs">
-        <Label>Saldo:</Label>
-        <Balance>{numberWithSpaces(account.balance)},- NOK</Balance>
-      </Card.Section>
-      <Card.Section m="xs">
-        <Label>Kortnummer:</Label>
-        <CardInputContainer>
-          <input
-            value={cardUuid}
-            disabled={!editable}
-            onChange={evt => setCardUuid(evt.target.value)}
-            onBlur={toggleEditable}
-          />
-          <EditIcon onClick={toggleEditable} />
-        </CardInputContainer>
-      </Card.Section>
+    <Card withBorder className={classes.balanceCard}>
+      <Group position={'right'}>
+        <Title
+          order={4}
+          variant={'gradient'}
+          gradient={{ from: 'samfundet-red.0', to: 'samfundet-red.2', deg: 30 }}
+        >
+          Socibanken
+        </Title>
+      </Group>
+      <Container px={'md'} py={'sm'}>
+        <Group position={'apart'}>
+          <Card.Section p={'lg'}>
+            <Badge color={'samfundet-red'} size={'sm'}>
+              Saldo
+            </Badge>
+            <Title align="center" weight={'lighter'} color={'gray.0'}>
+              {numberWithSpaces(account.balance)} kr
+            </Title>
+          </Card.Section>
+          <Button
+            component={Link}
+            to={'/economy/deposits/create'}
+            variant={'light'}
+            color={'samfundet-red'}
+          >
+            <IconCash />
+            <IconPlus />
+          </Button>
+        </Group>
+        <Group position={'center'}>
+          <Card.Section>
+            <Text align="center">Kortnummer</Text>
+            <Group position="center">
+              <TextInput
+                value={cardUuid}
+                disabled={!editable}
+                onChange={evt => setCardUuid(evt.target.value)}
+                onBlur={toggleEditable}
+              />
```

Comment:
Boyscout coder moment, since you're already here feel free to change from react-hot-toast to Mantine `showNotification`

Reply:
fisked